packages/dso-toolkit/components/Componenten/label/_preview.njk

```diff
@@ -9,8 +9,27 @@
 
   <body>
     <div class="toolkit-example-container container">
-      <div class="row">
-        {{ yield | safe }}
+      <div class="toolkit-example-wrapper">
+        <div class="toolkit-example-text">
+          <h2>Labels</h2>
+        </div>
+
+        <div class="toolkit-example">
+          <div class="row">
+            <div class="col-xs-4">
+              <h3>Label default</h3>
+            </div>
+            <div class="col-xs-4">
+              <h3>Label met button</h3>
+            </div>
+            <div class="col-xs-4">
+              <h3>Label small</h3>
```

Comment:
Laten we hier dan ook overstappen op de compact benaming

Reply:
Mee eens